lib/ansiblelint/__init__.py

```diff
@@ -162,7 +161,7 @@ def run(self, playbookfile, tags=set(), skip_list=frozenset()):
         matches = list()
 
         try:
-            with codecs.open(playbookfile['path'], mode='rb', encoding='utf-8') as f:
+            with open(playbookfile['path'], mode='r', encoding='utf-8') as f:
```

Comment:
Maybe it's better to keep binary mode?

Reply:
That is not an option, removal of "b" is mandatory as some supported versions of Python raise exception. In fact it makes no sense to use an encoding with a binary, these being exclusive. You can say that allowing it was a bug in codecs.

Reply:
Ah, okay. I was concerned by us changing the type of the result variable and how this can affect the rest of the code.